.github/workflows/build-test.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     - uses: actions/checkout@v3
     - name: Unix (FreeBSD) test
       id: Test
-      uses: vmactions/freebsd-vm@v0.1.6
+      uses: vmactions/freebsd-vm@v0.2.6
```

Comment:
The suggestion was to use `vm@0` so that we always use the newest major version otherwise we will encounter this problem again.

But you do not need to handle it in this PR, #1625 already fixes it